### PR TITLE
allow for optional not creating pvc in galaxy deployment

### PR DESCRIPTION
**Pull request by andersla**
It is not tested yet - but it looks ok or what do you think?

### CODE REVIEW

File: galaxy-simple/templates/galaxy_pvc.yaml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+{{ if .Values.galaxy_create_pvc }}
```
Comment (pcm32) [inline galaxy-simple/templates/galaxy_pvc.yaml:1]: You need to close this if at the end of what you want to enclose, with an `{{ end }}`

File: galaxy-simple/values.yaml [hunk 2]
```diff
@@ -39,6 +39,7 @@ galaxy_node_port_exposed: 30700
 domain: "mydomain.dev" 
 
 # Storage request
+galaxy_create_pvc: true
```
Comment (pcm32) [inline galaxy-simple/values.yaml:42]: I haven't really used booleans in go template, not sure how this will go ;-)

**Review by pcm32 — CHANGES_REQUESTED** [review]
Many thanks for this @andersla ! I think it will come in handy for some use cases. In particular, you probably want to make these changes as well on galaxy-postgre-chart, which is the one we use for the serious deployments, galaxy-simple is mostly for developer usage. Thanks!

File: galaxy-simple/values.yaml (hunk 2 above)
Comment (andersla) [reply galaxy-simple/values.yaml:42]: @pcm32 Worked ok with default settings - nothing changed. Galaxy up - no error in galaxy-log. Also:  
`shared-volume   195G       RWX           Retain          Bound     default/galaxy-pvc             13m`

**Comment by andersla** [comment]
I'm so stupid sometimes:) thats where
      they should be - comming soon:)
      
      On 2017-02-28 15:50, Pablo Moreno wrote:
    
    
      @pcm32 requested changes on this pull request.
      Many thanks for this @andersla
        ! I think it will come in handy for some use cases. In
        particular, you probably want to make these changes as well on
        galaxy-postgre-chart, which is the one we use for the serious
        deployments, galaxy-simple is mostly for developer usage.
        Thanks!
      
      In galaxy-simple/templates/galaxy_pvc.yaml:
      > @@ -1,3 +1,4 @@
+{{ if .Values.galaxy_create_pvc }}

      You need to close this if at the end of what you want to
        enclose, with an {{ end }}
      
      In galaxy-simple/values.yaml:
      > @@ -39,6 +39,7 @@ galaxy_node_port_exposed: 30700
 domain: "mydomain.dev" 
 
 # Storage request
+galaxy_create_pvc: true

      I haven't really used booleans in go template, not sure how
        this will go ;-)
      —
        You are receiving this because you were mentioned.
        Reply to this email directly, view
          it on GitHub, or mute
          the thread.
      
        
          
          
        
        
      
      {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/pcm32/galaxy-helm-charts","title":"pcm32/galaxy-helm-charts","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/pcm32/galaxy-helm-charts"}},"updates":{"snippets":[{"icon":"PERSON","message":"@pcm32 requested changes on #2"}],"action":{"name":"View Pull Request","url":"https://github.com/pcm32/galaxy-helm-charts/pull/2#pullrequestreview-24254782"}}}

**Comment by pcm32** [comment]
Thanks @andersla !

**Review by pcm32 — APPROVED** [review]

**Pull request merged**